config/fcos/v1_6_exp/schema.go

```diff
@@ -32,6 +32,7 @@ type BootDevice struct {
 
 type BootDeviceLuks struct {
 	Discard   *bool       `yaml:"discard"`
+	Device    *string     `yaml:"device-s390x"`
```

Comment:
Why are we adding a new field?

Reply:
Correct me if am wrong. 
I added the `Device` for s390x disk as new field. So that when user generate ignition from butane config they must provide s390x device like /dev/dasda or /dev/sda. Else it use the label which we do not expect for s390x.